parm/soca/letkf/letkf.yaml.j2

```diff
@@ -11,8 +11,8 @@ background:
   members from template:
     template:
       date: '{{ ATM_WINDOW_MIDDLE }}'
-      ocn_filename: enkfgdas.ocean.t06z.inst.f009.nc
-      ice_filename: enkfgdas.ice.t06z.inst.f009.nc
+      ocn_filename: "enkf{{ RUN }}.ocean.t{{ gcyc }}z.inst.f006.nc"
+      ice_filename: "enkf{{ RUN }}.ice.t{{ gcyc }}z.inst.f006.nc"
```

Comment:
I could be wrong but I don't think there is such a thing as `enkfgfs`. @CoryMartin-NOAA ?

Reply:
for backgrounds no, but for output there sure is!

Reply:
Regardless this should be `"enkf{{ GDUMP }}"`

Reply:
Should all of the backgrounds, ensemble and deterministic, have `{{ GDUMP }}`?

Reply:
yes

Reply:
You can't use RUN because the background for both the GFS and GDAS will be GDAS